### DISCUSSION TITLE
Adding support for more devices...

### DISCUSSION THREAD

**Discussion by iklein99** [iklein99/homebridge-smartthings#57] [Announcements]
Hi everyone.  I hope you are enjoying this plugin.  I'm happy that so many of you are using my plugin and would like to continue to add support for more devices with your help.

I need your help with 2 things:

First, as I add some devices which I may not have, I may have to ask some of you to help me test.

Second, continuing to support an improve this plugin is proving time consuming and I've been doing this with no compensation so far (except knowing that I'm helping so many of you out).   However, I'd like to ask that if you value this plugin and would like to help to keep it going, please contribute whatever you feel is fair.  This will also allow me to purchase more device type to test with so I can continue to enhance the plugin.

You can Venmo me at **Ira-Klein-3**.  

Thanks!

**Comment by iklein99** [comment]
Me again...  I have added a link to help fund the ongoing development of this project.  I hope if you get value from it, you will consider a contribution.  At some point, I may require some form of payment to run new updates to the plugin, but I'd rather not do that.

**Comment by coryabooth** [comment]
Can you add a "Donation" link here?
Or is this specific to the UI plugin...

![image](https://user-images.githubusercontent.com/5266177/198670108-85c1ec6e-5967-45fc-a539-2e18d8726cb1.png)

**Comment by iklein99** [comment]
I have to get the plugin verified.  I’m in the queue to do that now.  However, there is a sponsor link in GitHub:  https://github.com/iklein99/homebridge-smartthings <https://github.com/iklein99/homebridge-smartthings>
